packages/frontend/src/pages/admin/abuses.vue

```diff
@@ -47,27 +47,89 @@ SPDX-License-Identifier: AGPL-3.0-only
 				</div>
 			</div>
 		</div>
+		<div v-else>
+			<div class="_gaps">
+				<MkFolder ref="folderComponent">
+					<template #label><i class="ti ti-plus" style="margin-right: 5px;"></i>{{ i18n.ts.createNew }}</template>
+					<MkAbuseReportResolver v-model="newResolver" :editable="true">
+						<template #button>
+							<MkButton primary :class="$style.margin" @click="create">{{ i18n.ts.create }}</MkButton>
+						</template>
+					</MkAbuseReportResolver>
+				</MkFolder>
+				<MkPagination v-slot="{items}" ref="resolverPagingComponent" :pagination="resolverPagination">
+					<MkSpacer v-for="resolver in items" :key="resolver.id" :marginMin="14" :marginMax="22" :class="$style.resolverList">
+						<MkAbuseReportResolver v-model="editingResolver" :data="(resolver as any)" :editable="editableResolver === resolver.id">
+							<template #button>
+								<div v-if="editableResolver !== resolver.id">
+									<MkButton primary inline :class="$style['button-margin']" @click="edit(resolver.id)"><i class="ti ti-pencil"></i> {{ i18n.ts.edit }}</MkButton>
```

Comment:
```suggestion
									<MkButton primary inline :class="$style.button-margin" @click="edit(resolver.id)"><i class="ti ti-pencil"></i> {{ i18n.ts.edit }}</MkButton>
```
Productionビルドでリストが見えなくなる問題がありました。この修正で直ります。

Reply:
これでリストが見えなくなる問題は解決しますが、スタイルがうまく反映されなかったのとVolarがエラーを出していたので別途変更をコミットしました🙏